x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_params_schema.ts

```diff
@@ -37,6 +37,9 @@ const signalSchema = schema.object({
   severity: schema.string(),
   severityMapping: schema.nullable(schema.arrayOf(schema.object({}, { unknowns: 'allow' }))),
   threat: schema.nullable(schema.arrayOf(schema.object({}, { unknowns: 'allow' }))),
+  threshold: schema.maybe(
```

Comment:
I think is better to declare it as `nullable` than `maybe`.

Reply:
I think we want to keep this as a `maybe` since a `null` value here doesn't intrinsically mean anything. This is also the pattern used with the other rule `type` dependent fields like `machineLearningJobId`.